packages/new-polymath-issuer/src/components/DividendList/Presenter.tsx

```diff
@@ -13,55 +13,57 @@ import * as sc from './styles';
 export interface Props {
   dividends: types.DividendPojo[];
   securityTokenSymbol: string;
+  checkpointIndex: number;
 }
 
 export const DividendListPresenter = ({
   securityTokenSymbol,
   dividends,
-}: Props) => (
-  <List>
-    {dividends.length ? (
-      <Fragment>
-        {dividends.map(dividend => (
-          <li key={dividend.uid}>
-            <DividendCard
-              dividend={dividend}
-              securityTokenSymbol={securityTokenSymbol}
+  checkpointIndex,
+}: Props) => {
+  const newDividendUrl = `/securityTokens/${securityTokenSymbol}/checkpoints/${checkpointIndex}/new`;
```

Comment:
Shouldn't this be `/securityTokens/${securityTokenSymbol}/checkpoints/${checkpointIndex}/dividends/new`?

Reply:
True

Reply:
```suggestion
  const newDividendUrl = `/securityTokens/${securityTokenSymbol}/checkpoints/${checkpointIndex}/dividends/new`;
```